cli/helm.go

```diff
@@ -15,32 +15,32 @@ import (
 // and should not be trusted.
 var version = "0.1.0"
 
-const globalUsage = `The Kubernetes package manager
+const globalUsage = `Helm Classic - The original Kubernetes package manager
```

Comment:
You could just keep it as `The Kubernetes Package Manager` or just `A Kubernetes Package Manager`.